content/privacy-policy/index.md

```diff
@@ -3,8 +3,7 @@ title: "Privacy Policy"
 date: 2024-05-16T16:04:57-04:00
 draft: false
 ---
-
-# Privacy Policy
+## Privacy Policy
```

Comment:
This can still be an `h1` (using `#`) instead of an `h2` (`##`) since it is the page header